projects/developer/src/app/theme/dark-theme.ts

```diff
@@ -94,6 +94,19 @@ export const darkTheme: Theme = {
         '--pre-border': '#777',
         '--pre-on': '#f5f5f5',
         '--panel-header-hover': '#333',
-        '--panel-content-hover': '#3d4952'
+        '--panel-content-hover': '#3d4952',
+        '--info-main': 'hsl(194,66%,61%)',
```

Comment:
could you actually convert these to hex for now? the rest of the file is in hex, so it's easier for future debugging if things were consistent. for new projects we can use hsl.

Reply:
Sure, no problem.